app/controllers/policies_controller.rb

```diff
@@ -72,7 +72,23 @@ def customer_list
   end
 
   def category_fields
-    @category =params[:category]
+    @category = params[:category]
+    @company = Category.find_by(name: params[:category]).companies
```

Comment:
@KaranBamaniya-BTC : Category.find_by(name: params[:category]). is common in the below lines